### PR TITLE
update lots of python tools and add a pip toolfile

### PR DESCRIPTION
**Pull request by davidlange6**

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: py2-pip-toolfile.spec [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+%prep
+
+%build
+
+%install
+
+mkdir -p %{i}/etc/scram.d
+cat << \EOF_TOOLFILE >%{i}/etc/scram.d/py2-pip.xml
+<tool name="py2-pip" version="@TOOL_VERSION@">
+  <info url="https://pypi.python.org/pypi/pip
```
Comment (smuzaffar) [inline py2-pip-toolfile.spec]: @davidlange6 , you are missing the ending ```>``` here

Comment (davidlange6) [reply py2-pip-toolfile.spec]: indeed
whoops

**Comment by smuzaffar** [comment]
please test

**Comment by davidlange6** [comment]
please test

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
@davidlange6 , looks like new version of py2-requests requires urllib3. Can you please it ?
```
>python -c 'import requests'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/var/build/muz/u/w/slc6_amd64_gcc530/external/py2-pippkgs/5.0-cms/lib/python2.7/site-packages/requests/__init__.py", line 43, in <module>
    import urllib3
ImportError: No module named urllib3
```

**Comment by davidlange6** [comment]
ok, I'll start over.. i messed up my area.

**Pull request closed without merging**